### PR TITLE
WIP: Add multilingual support for issue #9

### PR DESCRIPTION
**Pull request by jdillard**

### CODE REVIEW

**Comment by humitos** [comment]
Hi @jdillard! I'm starting to work on an implementation of https://github.com/rtfd/readthedocs.org/issues/557 and I'm making some decisions. I wanted to know if you are still working on this project and/or PR.

I _suppose_ we are not going to install it by default on each build on Read the Docs, thought; but maybe suggest to our users.

So, no urgent, no needed, no nothing. Just asking about its status. Thanks a lot!

**Comment by jdillard** [comment]
Hey @humitos, that's great news! I can test and merge the two open PRs after work today.

I should be available if you plan on implementing it soonish and something comes up.

**Comment by humitos** [comment]
I started with the general implementation for all the projects at https://github.com/rtfd/readthedocs.org/pull/5122 --I'd appreciate your feedback.

I have to decide how to do the other case: users generating their own sitemaps and RTD serving them under a global one with `sitemapindex` tags. Also, if you have more feedback about this case, please feel free to comment the PR or the issue itself.

File: README.rst [hunk 1]
```diff
@@ -24,6 +24,31 @@ of your documentation. For example::
...
+config value in the generated URL.
+
+The end result is something like the following for each language/version build::
+
+  <?xml version="1.0" encoding="utf-8"?>
+  <urlset xmlns:xhtml="http://www.w3.org/1999/xhtml" xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+    <url>
+      <loc>https://my-site.com/docs/en/latest/index.html</loc>
+      <xhtml:link href="https://my-site.com/docs/es/latest/index.html" hreflang="es" rel="alternate"/>
+      <xhtml:link href="https://my-site.com/docs/fr/latest/index.html" hreflang="fr" rel="alternate"/>
```
Comment (humitos) [inline README.rst]: I'm not sure if this is just in the example or also in the code, but Google says that the original page from `<loc>` has to be added as an `alternate` also. So, I think there could be a bug here.

> Each language version must list itself as well as all other language versions.

from https://support.google.com/webmasters/answer/189077

and the example shown,

```xml
  <url>
    <loc>http://www.example.com/english/page.html</loc>
    <xhtml:link 
               rel="alternate"
               hreflang="de"
               href="http://www.example.com/deutsch/page.html"/>
    <xhtml:link 
               rel="alternate"
               hreflang="de-ch"
               href="http://www.example.com/schweiz-deutsch/page.html"/>
    <xhtml:link 
               rel="alternate"
               hreflang="en"
               href="http://www.example.com/english/page.html"/>
  </url>
```

Comment (jdillard) [reply README.rst]: Great catch! I fixed it in 1ca5bbad8adc65f61449b794f6e2ede603db4e38

**Pull request merged**